html-css/week2/style.css

```diff
@@ -0,0 +1,190 @@
+:root {
+  --white: #ffffff;
+  --lith-gray: #eae8e8;
+  --gray: #808080;
+  --dark-gray: #454444;
+  --black: #000000;
+  --purple: #ab5cdb;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: 'Roboto', sans-serif;
+  color: var(--black);
+}
+
+body {
+  min-width: 20rem;
+}
+
+p {
+  color: var(--gray);
+  margin-bottom: 0.625rem;
+}
+
+h1,
+h2,
+h3 {
+  text-transform: uppercase;
+  margin-bottom: 0.625rem;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  padding: 1.25rem 1.25rem;
+}
+
+.title_section-grey {
+  background-color: var(--lith-gray);
+}
+
+p.article-title {
+  text-transform: lowercase;
+}
+
+.header-nav__items {
+  text-transform: lowercase;
+}
+
+.header-nav__item a {
+  margin-top: 0.625rem;
+  color: inherit;
+}
+
+.header-nav__item a:hover,
+.header-nav__item a:active {
+  color: var(--gray);
+}
+
+.img-container img {
+  width: 100%;
+  height: 30%;
+}
+
+.logo-box {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+iframe {
+  margin-top: 1.25rem;
+  width: 90%;
+  height: 70%;
+}
+
+footer {
+  background-color: var(--purple);
+}
+
+.footer h3,
+.footer p,
+.footer a {
+  color: var(--white);
+}
+
+@media (min-width: 40rem) {
+  h1 {
```

Comment:
With media quires, it's better to stick to `px` and use the general breakpoints of different devices - https://css-tricks.com/snippets/css/media-queries-for-standard-devices/